app/i18n/en/sub.php

```diff
@@ -67,7 +67,7 @@
 		),
 		'description' => 'Description',
 		'empty' => 'This feed is empty. Please verify that it is still maintained.',
-		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
+		'error' => 'This feed has encountered a problem. If this problem persists, If this problem persists, please verify that it is still reachable.',
```

Comment:
```suggestion
		'error' => 'This feed has encountered a problem. If this problem persists, please verify that it is still reachable.',
```
I'm not completely happy about the non-typo repetition of problem though. Maybe something like this?
```suggestion
		'error' => 'This feed has encountered a problem. If this situation persists, please verify that it is still reachable.',
```

Reply:
https://github.com/FreshRSS/FreshRSS/pull/7049/commits/6529137f0c8ea394e87a981fe72b678df4c0f098